rllib/env/wrappers/pettingzoo_env.py

```diff
@@ -43,7 +43,7 @@ class PettingZooEnv(MultiAgentEnv):
                 [0, 0, 0],
                 [0, 0, 0]]], dtype=uint8)
         }
-        >>> obs, rewards, dones, truncateds, infos = env.step({
+        >>> obs, rewards, terminateds, truncateds, infos = env.step({
```

Comment:
Minor comment - the gymnasium project is changing our language in the vector environments to use `terminations` rather than `terminateds` for better english. Feel free to change or not